Weather/Views/DailyWeatherView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DailyWeatherView.swift
+//  Weather
+//
+//  Created by Oleshkevich Artem on 26.07.22.
+//
+
+import SwiftUI
+
+struct DailyWeatherView: View {
+    @ObservedObject var cityViewModel: CityViewViewModel
+    
+    var body: some View {
+        VStack {
+            Text(NSLocalizedString("Daily weather", comment: ""))
```

Comment:
There's no need to use `NSLocalizedString` in SwiftUI, you can just write `Text("Daily weather")` where `"Daily weather"` is your localisation key or you should use `LocalizedStringKey` if it won't work